cfe_internal/enterprise/federation/federation.cf

```diff
@@ -319,10 +319,9 @@ bundle agent clean_when_off
       "has_cftransport_fcontext" expression => returnszero("$(paths.semanage) fcontext -l | grep $(home)", "useshell");
 
   commands:
-      am_superhub::
```

Comment:
So, with this class guard removed both remote_hubs and federated_reporting_settings tables will be truncated on both feeders and the superhub. Is that what we want?

Reply:
yes, right, we need to truncate them as part of the cleanup task

Reply:
on both

Reply:
OK, it smells a bit funny to me to truncate tables every execution.